src/reifyhealth/specmonstah/core.cljc

```diff
@@ -237,18 +249,35 @@
         (lg/add-edges [ent-name related-ent-name])
         (lat/add-attr ent-name related-ent-name :relation-attrs (conj (or ids #{}) id)))))
 
-(defn related-ents
-  "Returns all related ents for an ent's relation-attr"
-  [{:keys [schema data] :as db} ent-name ent-type relation-attr related-ent-type query-term]
-  (let [{:keys [relations constraints]}   (ent-type schema)
-        attr-constraints                  (relation-attr constraints)
-        coll-attr?                        (coll-relation-attr? db ent-name relation-attr)
-        {:keys [refs bind]}               (and query-term
-                                               (-> (s/conform ::query-term query-term)
-                                                   second
-                                                   :query-opts))
-        [qr-constraint [qr-type qr-term]] (relation-attr refs)]
+(defn conformed-query-opts
+  "These conformed query opts allowo us to 1) validate the query term
```

Comment:
allowo! sounds like a harry potter spell

Reply:
lol

Reply:
i have in fact been re-watching the harry potter movies. makes me think of minikanren, which seems to have bought entirely into the harry potter naming scheme